changelog/28669.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+core: Extend panic fix for help request URL paths without /v1/ prefix provided by [#14704](https://github.com/hashicorp/vault/pull/14704).
```

Comment:
This doesn't tell me a lot -- if I was reading a changelog I might not really know what this meant (and it'd make me need to dig up a two year old PR for context). Maybe something like:
`Fixed panic seen when performing help requests without `/v1/` in the URL`